selenium/spec/trees_spec.rb

```diff
@@ -121,16 +121,29 @@
     $driver.find_element(:id => js_node(@a2).li_id)
       .find_element(:css => "i.jstree-icon").click
     sleep(5)
-
-    $driver
-      .find_element(:id => js_node(@a2).li_id)
-      .find_element(:id => js_node(@a3).li_id).find_element(:css => "span.title-column").text.should match(/Resource Component/)
-
+    
+    expect {
+
+      $driver
+        .find_element(:id => js_node(@a2).li_id)
+        .find_element(:id => js_node(@a3).li_id).find_element(:css => "span.title-column").text.should_not match(/Resource Component/)
+    }.to raise_error
+    
     # if we refresh the parent should now be open
     $driver.navigate.refresh
-    $driver
-      .find_element(:id => js_node(@a2).li_id)
-      .find_element(:id => js_node(@a3).li_id).find_element(:css => "span.title-column").text.should match(/Resource Component/)
+    
+    expect  { 
+      $driver
+        .find_element(:id => js_node(@a2).li_id)
+        .find_element(:id => js_node(@a3).li_id).find_element(:css => "span.title-column").text.should_not match(/Resource Component/)
+    }.to raise_error
```

Comment:
@cfitz - I'm a little confused by this. Are we just expecting .find_element(:id => js_node(@a2).li_id) to fail?